tests/e2e/specs/stories-editor/page.js

```diff
@@ -218,4 +248,102 @@ describe( 'Story Page', () => {
 		const previewPage = await openPreviewPage( editorPage, 'amp-story' );
 		await previewPage.waitForXPath( '//amp-video[contains(@aria-label, "Hello World")]' );
 	} );
+
+	it( 'should pass global story settings to a new story', async () => {
+		const documentPanelSelector = 'button.edit-post-sidebar__panel-tab[data-label="Document"]';
+		const storySettingsPanelSelector = '.amp-story-settings-panel';
+		const storySettingsAdvanceAfterSelector = '.components-panel__body.amp-story-settings-panel .components-select-control__input:nth-of-type(1)';
+		const storySettingsAdvanceAfterDurationSelector = '.components-panel__body.amp-story-settings-panel [aria-label="Time in seconds"]';
+
+		await page.click( documentPanelSelector );
+		await page.waitForSelector( storySettingsPanelSelector );
+
+		await page.click( storySettingsPanelSelector );
+		await page.waitForSelector( storySettingsAdvanceAfterSelector );
+
+		expect( await page.evaluate( ( selector ) => {
```

Comment:
Could we make the following into the helper function?
```
const getValueBySelet = ( el ) => {
         return await page.evaluate( ( selector ) => {
	           return document.querySelector( selector ).value;
	}, el ) );
}
```
Same code seems to be being reused a couple of times.

Reply:
You could do the same with 
```
		await page.evaluate( ( selector ) => {
			document.querySelector( selector ).value = '';
		}, secondsSelector );
```

Reply:
Done and added to the test file. Doesn't seem to be universal enough use-case to warrant its own helper.